### ISSUE TITLE
Whether implement support vector regression?

### ISSUE THREAD

**Issue by Spiridempt** [Jiaolong/ASVM-Linear#1]
Does this implementation contain support vector regression? Specifically, can I put float numbers in <label> and predict float numbers?

**Comment by Jiaolong** [comment]
No, this implementation only supports multi-class classification problem, e.g., http://www.linearsvm.com/.
